synapse/module_api/__init__.py

```diff
@@ -425,6 +425,15 @@ def register_user(
             )
         )
 
+    def get_user_total_message_count(self, user_id):
+        """Counts the total number of messages, sent by a user.
+        Args:
+            user_id (str): full canonical @user:id
+        Returns:
+            defer.Deferred[int] : The number of messages.
```

Comment:
On the docstring can you:

* add an empty line between each section
* add the version of Synapse the method will be shipped with - in this case it will likely be 1.48.0

Feel free to refer to other docstrings in the file for examples.